app/(backend)/dtos/users.dto.ts

```
@@ -0,0 +1,63 @@
+      .transform((val) => parseInt(val ?? "10"))
+      .refine((val) => val > 0, { message: "Limit must be greater than 0" }),
+
+    role: z
+      .string()
+      .optional()
+      .transform((val) => val?.toUpperCase())
+      .refine((val) => !val || ["BUSINESS_ANALYST", "ADMIN"].includes(val), {
+        message: "Role must be either BUSINESS_ANALYST or ADMIN",
+      }) as z.ZodType<"BUSINESS_ANALYST" | "ADMIN" | undefined>,
+  });
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)